.github/workflows/test.yaml

```
@@ -0,0 +1,22 @@
+name: Docker
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v4
```
v4 is latest https://github.com/actions/checkout#checkout-v4